### PR TITLE
chore: Uploading ccd definitions into preview env on demand

### PR DESCRIPTION
**Pull request by tomaszpowroznik**
https://build.platform.hmcts.net/job/HMCTS_FPL/job/fpl-ccd-configuration/job/PR-2125/3/artifact/ccd-release-config/

To support feature toggles on preview env

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,6 +153,41 @@ then
 curl http://localhost:9210/care_supervision_epo_cases-000001/_search
 ```
 
+## Uploading ccd definition into PR environment
+On PR env following ccd definition files are generated and stored as jenkins job artefacts:
+- ccd-fpl-preview-<PR-ID>-toggle-on.xlsx (uploaded automatically by jenkins)	
+- ccd-fpl-preview-<PR-ID-toggle-off.xlsx
```
Comment (lewisbirks) [inline README.md:159]:
```suggestion
- ccd-fpl-preview-<PR-ID>-toggle-off.xlsx	
```

File: README.md [hunk 2]
```diff
@@ -153,6 +153,41 @@ then
 curl http://localhost:9210/care_supervision_epo_cases-000001/_search
 ```
 
+## Uploading ccd definition into PR environment
+On PR env following ccd definition files are generated and stored as jenkins job artefacts:
+- ccd-fpl-preview-<PR-ID>-toggle-on.xlsx (uploaded automatically by jenkins)
```
Comment (lewisbirks) [inline README.md:158]: the rich diff doesn't seem to be displaying anything here however the markdown interpreter in intellij is 🤔 
image
image

**Review by lewisbirks — COMMENTED** [review]
looks good some very minor things, would it be worth mentioning that you need to be on vpn as well

File: README.md (hunk 2 above)
Comment (tomaszpowroznik) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (tomaszpowroznik) [reply README.md]: done

**Review by lewisbirks — APPROVED** [review]

**Pull request merged**